.github/workflows/stale.yml

```
@@ -0,0 +1,24 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: '30 * * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        debug: true
```

@yubiuser my understanding is that it would be in the log output of the Action run, which will show up here: https://github.com/pi-hole/pi-hole/actions

---

See also https://github.com/actions/stale#debugging

---

Intentionally set to debug so that we have an idea on the first run what the result would be when it goes to live mode.

---

`debug` or `debug-only`?

---

```suggestion
        debug-only: true
```

I think that may have changed between v3 and v4.